### PR TITLE
Move version chip to footer

### PR DESCRIPTION
**Pull request by ehab-hassan**
### Description

the version chip overlaps with the sidebar extending so ll move it to footer 

### Changes

Move version chip to footer

image

### Related Issues

#3514

### Tested Scenarios

A list of scenarios tried to match the deliverables

### Documentation PR

For UI changes, Please provide the Documetation PR on [info_grid](https://github.com/threefoldtech/info_grid)

#### Preliminary Checks:

- Does it completely address the issue linked?
- What about edge cases?
- Does it meet the specified acceptance criteria?
- Are there any unintended side effects?
- Does the PR adhere to the team's coding conventions, style guides, and best practices?
- Does it integrate well with existing features?
- Does it impact the overall performance of the application?
- Are there any bottlenecks or slowdowns?
- Has it been optimized for efficiency?
- Has it been adequately tested with unit, integration, and end-to-end tests?
- Are there any known defects or issues?
- Is the code well-documented?
- Are changes to documentation reflected in the code?

#### UI Checks:

- If a UI design is provided/ does it follow it?
- Does every button work?
- Is the data displayed logical? Is it what you expected?
- Does every validation work?
- Does this interface feel intuitive?
- What about slow network? Offline?
- What if the gridproxy/graphql/chain is failing?
- What would a first time user have a hard time navigating here?

#### Code Quality Checks:

- Code formatted/linted? Are there unused imports? .. etc
- Is there redundant/repeated code?
- Are there conditionals that are always true or always false?
- Can we write this more concisely?
- Can we reuse this code? If yes, where?
- Will the changes be easy to maintain and update in the future?
- Can this code become too complex to understand for other devs?
- Can this code cause future integration problems?

### Checklist

- [ ] Tests included
- [x] Build pass
- [ ] Documentation
- [ ] Code format and docstrings
- [x] Screenshots/Video attached (needed for UI changes)

### CODE REVIEW

File: packages/playground/src/components/main_footer.vue [hunk 1]
```diff
@@ -82,7 +82,17 @@
         </v-col>
       </v-row>
       <v-divider class="my-3 w-25 mx-auto" />
-      <div class="text-center mb-12">{{ new Date().getFullYear() }} — ThreeFoldTech</div>
+      <div class="d-flex justify-center text-subtitle-2">
+        <div class="text-center mb-12">
+          &#169; {{ new Date().getFullYear() }} ThreeFoldTech <span class="mx-2">|</span>
+        </div>
+        <template v-if="version">
```
Comment (samaradel) [inline packages/playground/src/components/main_footer.vue]: why template outside the div ?

**Comment by samaradel** [comment]
Please hide the ```|``` and the version if it's not existed
image

**Review by samaradel — APPROVED** [review]

File: packages/playground/src/components/main_footer.vue [hunk 2]
```diff
@@ -82,7 +82,18 @@
         </v-col>
       </v-row>
       <v-divider class="my-3 w-25 mx-auto" />
-      <div class="text-center mb-12">{{ new Date().getFullYear() }} — ThreeFoldTech</div>
+
+      <div class="mb-12 d-flex justify-center text-subtitle-2">
+        <p>
+          &#169; {{ new Date().getFullYear() }} ThreeFoldTech
+          <span v-if="version !== 'No version to show'">
```
Comment (amiraabouhadid) [inline packages/playground/src/components/main_footer.vue]: might be a good idea to place 'No version to show' in a constant @AhmedHanafy725

Comment (ehab-hassan) [reply packages/playground/src/components/main_footer.vue]: the idea is to show the version to the user, if there is no version, there is no need to tell him that 
@AhmedHanafy725 what do you think?

Comment (ehab-hassan) [reply packages/playground/src/components/main_footer.vue]: > might be a good idea to place 'No version to show' in a constant @AhmedHanafy725

Done @amiraabouhadid can you check please

File: packages/playground/src/components/main_footer.vue [hunk 3]
```diff
@@ -96,10 +108,10 @@ import { AppThemeSelection } from "@/utils/app_theme";
 import { manual } from "@/utils/manual";
 
 const theme = useTheme();
-
+const version = process.env.VERSION as any;
```
Comment (amiraabouhadid) [inline packages/playground/src/components/main_footer.vue]: Why as any and not as string?

**Comment by amiraabouhadid** [comment]
@ehab-hassan hey I actually meant declaring 'no current version to show' as an app wide constant as it is used in multiple places, not as a message string in a single component

**Review by 0oM4R — APPROVED** [review]

**Pull request merged**